Makefile

```diff
@@ -0,0 +1,35 @@
+build: ## Build the backend Docker image
+	docker-compose build web
+
+up: ## Bring the backend Docker container up
```

Comment:
suggestion: `make start` would be more consistent with `make stop` (start/stop is easier to remember then up/stop)